.github/workflows/doc.yml

```
@@ -0,0 +1,44 @@
+name: Documentation
+on:
+  push: { branches: [ "main" ] }
+  pull_request: { branches: [ "main" ] }
+
+concurrency:
+  group: doc-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

Is there a reason to stick with ubuntu 20.04?
```suggestion
    runs-on: ubuntu-latest
```